### DISCUSSION TITLE
Symfony\Component\ErrorHandler\Error\FatalError: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Target [Illuminate\Contracts\Auth\Access\Gate] is not instantiable. in vendor/laravel/framework/src/Illuminate/Container/Container.php:1103

### DISCUSSION THREAD

**Discussion by Pascualsolutions** [wintercms/winter#1108] [Q&A] [answered]
Hello, i have this error in log i try to clear cache but this error always show up again.
Symfony\Component\ErrorHandler\Error\FatalError: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Target [Illuminate\Contracts\Auth\Access\Gate] is not instantiable. in /home/pascual/Desktop/wintercmsTest/winterCMSSQlitenovi/vendor/laravel/framework/src/Illuminate/Container/Container.php:1103

Can u help.
Alex

**Comment by mjauvin** [comment]
What version of Winter are you using? Also, does the error happen if you disable all your plugins ?

**Reply by Pascualsolutions** [reply]
My version is 1.2.5, i disabled all plugin (i have one Blog) but its the same error again.

**Comment by mjauvin** [comment]
That's odd. So you're saying this happens when you run `artisan cache:clear` ?

**Comment by LukeTowers** [comment, marked as answer]
This is happening because you are using the Laravel Extra Intellisense plugin for VSCode. Set LaravelExtraIntellisense.disableAuth in your vscode settings to true.

See https://github.com/amir9480/vscode-laravel-extra-intellisense/issues/66#issuecomment-1498828068
